templates/publisher/market.html

```diff
@@ -6,23 +6,31 @@
 
 {% block content %}
   <div id="main-content" class="u-no-margin--top">
-    <section class="p-strip is-shallow">
-      <div class="row">
-        <p><a href="/account">My snaps&nbsp;&rsaquo;</a></p>
-        <h1 class="u-no-margin--top">{{ title }}</h1>
-      </div>
-    </section>
-
-    <section class="metrics__nav-wrap p-strip is-shallow u-no-padding--top u-no-padding--bottom">
+    <section class="p-strip is-shallow u-no-padding--bottom">
       <nav class="p-tabs">
-        <ul class="p-tabs__list" role="tablist">
-          <li class="p-tabs__item" role="presentation">
-            <a href="/account/snaps/{{ snap_name }}/market" class="p-tabs__link" tabindex="0" role="tab" aria-selected="true">Market</a>
-          </li>
-          <li class="p-tabs__item" role="presentation">
-            <a href="/account/snaps/{{ snap_name }}/measure" class="p-tabs__link" tabindex="0" role="tab">Measure</a>
-          </li>
-        </ul>
+        {# This row and it's contents needs to be removed before going live #}
+        <div class="row">
+          <a href="/account" class="u-float--left p-muted-heading">&lsaquo; Snap list</a>
+        </div>
+        <div class="row u-no-margin--top">
+        <div class="row">
+          <div class="col-6">
+            <h3 class="u-float--left">{{ title }}</h3>
```

Comment:
If this really is an `<h3>`, what are the `<h1>` and `<h2>`?

Alternatively, if you’re just using `<h3>` because it happens to be the font size you want, isn’t that what CSS is for?

It wouldn’t surprise or concern me if headings on snapcraft.io were smaller than in standard Vanilla, for greater information density. But if so, let’s fix that in one place, rather than changing heading levels throughout the code.

Reply:
Well caught @matthewpaulthomas I wasn't thinking. Have fixed this here as this pr focuses on the publisher pages only... I think a bigger investigation for **all** headings across the site needs to be done before we commit to changing the Vanilla variables.